backend/organization/models/status.py

```diff
@@ -70,3 +70,86 @@ class Meta:
 
     def __str__(self):
         return "Project status  %s" % (self.name)
+
+
+def project_type_image_path(instance, filename):
+    return "project_type_icons/{}/{}".format(instance.id, filename)
+
+
+class ProjectTypes(models.Model):
+    name = models.CharField(
```

Comment:
Can users define custom project types? If yes, I would like to understand it better. But if its maintained by us then we do not need a database table. I think we can use constants or enums (https://docs.python.org/3/library/enum.html) You can store the icons as static files as well.

Reply:
Currently they don't and you're right we should use an enum for this. I was thinking this was more dynamic and easier to make changes for non-devs but these advantages don't outweight the disadvantage of using a database for static data at all. I guess I just went with the same concept we used for the old "Project Status". I'll make the necessary changes in the backend, thanks for pointing this out.

Would you still keep endpoints for this and just make them access the enum in the backend or additionally store the enum in the frontend? :thinking: